rc/base/autowrap.kak

```diff
@@ -37,9 +39,9 @@ def -hidden autowrap-cursor %{ eval -save-regs '/"|^@m' %{
 } }
 
 def autowrap-enable -docstring "Wrap the lines in which characters are inserted" %{
-    hook -group autowrap window InsertChar [^\n] autowrap-cursor
+    hook -group autowrap buffer InsertChar [^\n] autowrap-cursor
```

Comment:
I thought that hooks that modify the buffer would be better declared in the `buffer` scope, was I wrong ? Tbh I was never really clear on the difference between the `buffer` and `windows` scope (even if I get roughly what they refer to because of their name), maybe the scopes should have their own documentation page ?